backend/corpora/common/corpora_orm.py

```diff
@@ -1,189 +1,301 @@
+import enum
 import os
 import sys
 
 from sqlalchemy import (
-    create_engine,
     Boolean,
     Column,
+    create_engine,
     DateTime,
+    Enum,
     ForeignKey,
+    ForeignKeyConstraint,
     Integer,
     String,
     text,
-    BigInteger,
 )
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, sessionmaker
 
-pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))  # noqa
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
-from .corpora_config import CorporaDbConfig
+from common.corpora_config import CorporaDbConfig
 
 
 Base = declarative_base()
 deployment_stage = os.environ["DEPLOYMENT_STAGE"]
 
-# The in-memory SQLite database used in unit tests does not support now() SQL syntax
-DEFAULT_DATETIME = "2000-01-01 00:00:00" if deployment_stage == "test" else text("now()")
+DEFAULT_DATETIME = text("now()")
 
 
 class DBSessionMaker:
     def __init__(self):
-        connection = "sqlite:///:memory:" if deployment_stage == "test" else CorporaDbConfig().database_uri
-        self.engine = create_engine(connection)
+        self.engine = create_engine(CorporaDbConfig().database_uri)
         self.session_maker = sessionmaker(bind=self.engine)
 
     def session(self, **kwargs):
         return self.session_maker(**kwargs)
 
 
-class Project(Base):
-    __tablename__ = "project"
+class ProjectStatus(enum.Enum):
+    """
+    Describes a DbProject's status.
+    At most, one LIVE and one EDIT entry of a Project may exist at a time.
 
-    id = Column(String(64), primary_key=True)
-    title = Column(String(255))
-    label = Column(String(255))
-    description = Column(String(3000))
-    biosample_categories = Column(String(32))
-    development_stages = Column(String(150))
-    diseases = Column(String(1000))
-    cell_isolation_methods = Column(String(100))
-    cell_types = Column(String(150))
-    cell_count = Column(Integer)
-    paired_end = Column(String(16))
-    nucleic_acid_sources = Column(String(64))
-    input_nucleic_acid_molecules = Column(String(100))
-    publication_title = Column(String(200))
-    publication_doi = Column(String(32))
-    cxg_enabled = Column(Boolean)
-    created_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
-    updated_at = Column(DateTime(True), nullable=False, server_default=DEFAULT_DATETIME)
+    LIVE - a published and publicly viewable Project.
+    Edit - an open Submission, i.e an unpublished and non-public Project.
```

Comment:
Tiny nit: s/Edit/EDIT. to, as Trent says, "appease my OCD" 🤣

Reply:
Oh heh thought I fixed that. Good catch!